bodhi/server/templates/update.html

```diff
@@ -508,6 +542,31 @@ <h1>
             % else:
               <a id='revoke' class="btn btn-sm btn-danger"><span class="fa fa-fw fa-arrow-circle-left"></span> Revoke</a>
             % endif
+            % if request.registry.settings.get('test_gating.required') and not update.test_gating_passed:
```

Comment:
This is in the 'unpushed' side of a big if block, but don't forget that we may want to also be able to waive pushed updates too. I recommend pulling this out of the larger if block if feasible. One day I'm going to refactor the code here because I really find it messy (lots of repeated code).

Reply:
Hmm, can you help me recall what the use case is for waiving the updates that have been pushed?

Reply:
Hi @matt8754!

So "pushed" in Bodhi only means that it is in a repository (not that it is necessarily stable). Therefore, testing updates are still considered "pushed". I would assume that you would want developers to be able to waive updates that are in the testing state?

Reply:
Aha, I think I got it, thanks, :-D